invoice/src/main/java/org/killbill/billing/invoice/InvoiceDispatcher.java

```diff
@@ -609,6 +609,11 @@ private Iterable<UUID> getFilteredSubscriptionIdsFor_UPCOMING_INVOICE_DryRun(@Nu
                 .collect(Collectors.toUnmodifiableSet());
     }
 
+    private static boolean isPropertyReuseDraftSet(final Iterable<PluginProperty> inputProperties) {
+        // Hack to allow reusing a draft invoice when invoked through api
+        return Iterables.toStream(inputProperties)
+                        .anyMatch(p -> p.getKey().equals("KB_AUTO_INVOICING_REUSE_DRAFT") && ((String) p.getValue()).equalsIgnoreCase("true"));
```

Comment:
Nit: flip `.equals` for NPE safety.

Reply:
Fixed in https://github.com/killbill/killbill/pull/1776/commits/606139b76f26bf887a8e848af3b5c57596b167d4